README.md

```diff
@@ -19,6 +19,13 @@ SDK for Hyperliquid API trading with Python.
 ```bash
 pip install hyperliquid-python-sdk
 ```
+## Configuration 
+Generate and authorize a new API private key on https://app.hyperliquid.xyz/API
```

Comment:
good point

I've updated the content here to primarily point to the main wallet and optionally use an API wallet

@traderben im also thinking of moving this to use the [dotenv](https://pypi.org/project/python-dotenv/) package for managing the private key rather than holding it in the config file. Is that something you're supportive of?

Reply:
Thanks for updating the readme. I don't see the value in adding a dependency on dotenv, so I'd rather not add that change